sumologic/resource_sumologic_lookup_table.go

```diff
@@ -119,7 +122,7 @@ func resourceSumologicLookupTableRead(d *schema.ResourceData, meta interface{})
 
 	id := d.Id()
 	lookupTable, err := c.GetLookupTable(id)
-	fmt.Printf("##DEBUG## read lookup: %+v\n", lookupTable)
+	log.Printf("##DEBUG## read lookup: %+v\n", lookupTable)
```

Comment:
You can remove these and other print statements I added. I added them for debugging purpose.

Reply:
they only come if someone sets `TF_LOG` to `debug`. We can keep them for further debugging.